ci/kokoro/docker/build-in-docker-bazel.sh

```diff
@@ -197,12 +197,12 @@ if should_run_integration_tests; then
   source "${PROJECT_ROOT}/ci/kokoro/gcloud-functions.sh"
 
   trap delete_gcloud_config EXIT
+  activate_service_account_keyfile "${KOKORO_SETUP_KEY}"
```

Comment:
I don't know what this is related to, but I see the same error as in some of your presubmits.

Reply:
This "fix" was wrong. I think I fixed it correctly in my new PR: https://github.com/googleapis/google-cloud-cpp/pull/4288